src/components/ui/Chat/ChannelList/ChannelList.stories.tsx

```diff
@@ -6,18 +6,18 @@ import React, { useState } from 'react';
 import Flex from '../../Flex';
 import ChannelList from './';
 import ChannelItem from './ChannelItem';
-import ChannelListDocs from './ChannelList.mdx';
 import PopOverItem from '../../PopOver/PopOverItem';
+import ChannelListDocs from './ChannelList.mdx';
 
-export default {
-  title: 'UI Components/Chat',
-  parameters: {
-    docs: {
-      page: ChannelListDocs.parameters.docs.page().props.children.type
-    }
-  },
-  component: ChannelList
-};
+// export default {
```

Comment:
all of this is commented out!

Reply:
Yeah this is intentional. We don't want these stories published to storybook before the components are ready for release.